native_modules/RNDeviceModule.java

```diff
@@ -0,0 +1,436 @@
+package com.learnium.RNDeviceInfo;
```

Comment:
Why is this being copied into the project?

Reply:
If we're making changes locally, I'd prefer to fork the original project and use that instead so we can actually track the changes and make sure they're not broken when upgrading

Reply:
Elias and I decided on keeping the changes inside the project.  
If we want to fork the project that's cool.  Let me know what you guys decide

Reply:
Perhaps it will be best if we fork the projects.

fetch-blob: https://github.com/enahum/react-native-fetch-blob
device-info: https://github.com/enahum/react-native-device-info

Reply:
Cool deal, I'll make PRs to those forked projects